tests/integration/test_system_metrics/test.py

```diff
@@ -59,3 +59,25 @@ def test_readonly_metrics(start_cluster):
         node1.query("ATTACH TABLE test.test_table")
         assert_eq_with_retry(node1, "SELECT value FROM system.metrics WHERE metric = 'ReadonlyReplica'", "0\n", retry_count=300, sleep_time=1)
 
+def test_metrics_storage_buffer_size(start_cluster):
```

Comment:
Can you clarify how this test reproduces the problem?

Reply:
Clarification still is good to add